src/vcpkg/commands.autocomplete.cpp

```diff
@@ -33,15 +33,15 @@ namespace vcpkg::Commands::Autocomplete
     void perform_and_exit(const VcpkgCmdArguments& args, const VcpkgPaths& paths)
```

Comment:
I don't really know how to evaluate changes to this file, so if we're going to change it I think it needs e2e test(s) added. (Alternately, an API like `string get_autocomplete_stuff(StringView)` that can go into the unit test harness may make sense?)